manifests/c/ChemAxon/Marvin/22.3.0/ChemAxon.Marvin.locale.en-US.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: ChemAxon.Marvin
+PackageVersion: 22.3.0
+PackageLocale: en-US
+Publisher: ChemAxon
+Author: ChemAxon
+PackageName: ChemAxon Marvin Suite
+PackageUrl: https://chemaxon.com/products/marvin
+License: Copyright (C) ChemAxon

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate